### PR TITLE
Instruction to make a Route/KService visible only inside cluster.

### PR DESCRIPTION
**Pull request by tcnghia**
## Proposed Changes

- Update instructions to show how to make a Route/KService visible only inside a cluster.

### CODE REVIEW

File: serving/cluster-local-route.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Making your Routes local to the cluster.
+
+In Knative 0.3.x or later, all Routes with a domain suffix of
+`svc.cluster.local` will only be visible inside the cluster.
+
+This can be done by changing the `config-domain` config map as instructed
+[here](./using-a-custom-domain.md).
+
+In addition to that, you can set the label
```
Comment (carieshmarie) [inline serving/cluster-local-route.md]:
```suggestion
You can also set the label
```

File: serving/cluster-local-route.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+`serving.knative.dev/visibility=cluster-local` on your Route or KService to
+achieve the same effect.  For example, to make the Route `helloworld-go` in
+`default` namespace cluster local, run
+
+```shell
+kubectl label route helloworld-go serving.knative.dev/visibility=cluster-local
+```
+
+if such label was not already set when creating the Route.
```
Comment (carieshmarie) [inline serving/cluster-local-route.md]: Suggest incorporating this line in the previous sentence so it doesn't get lost:

```suggestion
For example, if you didn't set a label when you created the Route `helloworld-go` and you want to make it local to the `default namespace cluster, run:
```

File: serving/cluster-local-route.md (hunk 1 above)
Comment (tcnghia) [reply serving/cluster-local-route.md]: Done

File: serving/cluster-local-route.md (hunk 2 above)
Comment (tcnghia) [reply serving/cluster-local-route.md]: Done

**Comment by tcnghia** [comment]
@carieshmarie thanks for the suggestions.  I've made changes per requested.

File: serving/cluster-local-route.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Making your Routes local to the cluster.
```
[the remark this answers is not in the document]
Comment (tcnghia) [reply serving/cluster-local-route.md]: Done.

**Comment by tcnghia** [comment]
@samodell thanks for feedback.  Done now.

**Pull request merged**